datasets/ilist/README.md

```diff
@@ -0,0 +1,165 @@
+---
+task_categories:
+- text-classification
+multilinguality:
+- multilingual
+task_ids:
+- other
+languages:
+- hi
+- awa
+- bho
+- mag
+- bra
+annotations_creators:
+- unknown
+source_datasets:
+- original
+size_categories:
+- 10K<n<100K
+licenses:
+- unknown
+---
+
+# Dataset Card Creation Guide
+
+## Table of Contents
+- [Dataset Card Creation Guide](#dataset-card-creation-guide)
+  - [Table of Contents](#table-of-contents)
+  - [Dataset Description](#dataset-description)
+    - [Dataset Summary](#dataset-summary)
+    - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+    - [Languages](#languages)
+  - [Dataset Structure](#dataset-structure)
+    - [Data Instances](#data-instances)
+    - [Data Fields](#data-fields)
+    - [Data Splits](#data-splits)
+  - [Dataset Creation](#dataset-creation)
+    - [Curation Rationale](#curation-rationale)
+    - [Source Data](#source-data)
+      - [Initial Data Collection and Normalization](#initial-data-collection-and-normalization)
+      - [Who are the source language producers?](#who-are-the-source-language-producers)
+    - [Annotations](#annotations)
+      - [Annotation process](#annotation-process)
+      - [Who are the annotators?](#who-are-the-annotators)
+    - [Personal and Sensitive Information](#personal-and-sensitive-information)
+  - [Considerations for Using the Data](#considerations-for-using-the-data)
+    - [Social Impact of Dataset](#social-impact-of-dataset)
+    - [Discussion of Biases](#discussion-of-biases)
+    - [Other Known Limitations](#other-known-limitations)
+  - [Additional Information](#additional-information)
+    - [Dataset Curators](#dataset-curators)
+    - [Licensing Information](#licensing-information)
+    - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:**
+- **Repository:** [GitHub](https://github.com/kmi-linguistics/vardial2018)
+- **Paper:** [Link](https://www.aclweb.org/anthology/W18-3900/)
+- **Leaderboard:**
+- **Point of Contact:** linguistics.kmi@gmail.com
+
+### Dataset Summary
+
+This datasets is introduced in a task which aimed at identifying 5 closely-related languages of Indo-Aryan language family – Hindi (also known as Khari Boli), Braj Bhasha, Awadhi, Bhojpuri and Magahi. These languages form part of a continuum starting from Western Uttar Pradesh (Hindi and Braj Bhasha) to Eastern Uttar Pradesh (Awadhi and Bhojpuri) and the neighbouring Eastern state of Bihar (Bhojpuri and Magahi). For this task, participants were provided with a dataset of approximately 15,000 sentences in each language, mainly from the domain of literature, published over the web as well as in print.
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+ Hindi, Braj Bhasha, Awadhi, Bhojpuri and Magahi
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
```

Comment:
It would be great to have a sample here, in json format. Something like:
```json
{
    "language_id": "hi",
    "text": "some text in hindi"
}
```